src/components/tx-flow/flows/SignMessage/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Box, Typography } from '@mui/material'
 import SafeAppIconCard from '@/components/safe-apps/SafeAppIconCard'
 
 const APP_LOGO_FALLBACK_IMAGE = '/images/apps/apps-icon.svg'
-const APP_NAME_FALLBACK = 'Sign message off-chain'
+const APP_NAME_FALLBACK = 'Sign message'
```

Comment:
"off-chain" was being shown by default, regardless of whether it was on-/off-chain signing.